src/makie-axis.jl

```diff
@@ -213,19 +216,43 @@ function _point_iterator(plot::Plot)
     return _point_iterator(plot.plots)
 end
 
-function iterate_transformed(plot::Plot)
-    points = _point_iterator(plot)
-    t = Makie.transformation(plot)
-    model = Makie.model_transform(t)
-    trans_func = Makie.transform_func(t)
-    return Makie.iterate_transformed(points, model, to_value(get(plot, :space, :data)), trans_func)
+# function iterate_transformed(plot::Plot)
+#     points = _point_iterator(plot)
+#     t = Makie.transformation(plot)
+#     model = Makie.model_transform(t)
+#     trans_func = Makie.transform_func(t)
+#     return Makie.iterate_transformed(points, model, to_value(get(plot, :space, :data)), trans_func)
+# end
```

Comment:
This is probably replaceable by `boundingbox()` at some earlier stage, since that now always includes model and transform_func. (And it also includes markersize if `markerspace = space`)